src/utils/color-format-convertors.js

```diff
@@ -81,3 +81,59 @@ export function rgbToHsl(rgb) {
 export function hexToHsl(hex) {
   return rgbToHsl(hexToRgb(hex));
 }
+
+/**
+ * Convert css color form HSL to HEX format.
+ * @param {number} h  - hue
+ * @param {number} s  - saturation
+ * @param {number} l  - lightness
+ * @returns {string} hex  - css color in HEX format
+ */
+export function hslToHex(h, s, l) {
+  s /= 100;
```

Comment:
where is this copied from ?

Reply:
https://codepen.io/hillelkaltura/pen/NWOXMWM?editors=0110